frontend/src/api/api.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = 'https://jsonplaceholder.typicode.com';
+
+export const fetchSampleData = async () => {
```

Comment:
axiosa można skonfigurować tak że nie musisz wrzucać nawet returna

Reply:
wszystkie zapytania do api powinnt iść do odzielnego folderu nazwałbym go connectors i być pogrupowane

Reply:
jeżeli dobrze sonfigurujesz axiosa to zapytania możesz wykonywać tak export const fetchSampleData = async () => axios.get('/posts');